config/syslog.py

```diff
@@ -447,3 +457,63 @@ def delete(db, server_ip_address):
     except Exception as e:
         log.log_error("Failed to remove remote syslog logging: {}".format(str(e)))
         ctx.fail(str(e))
+
+
+@syslog.command("rate-limit-host")
+@click.option("-i", "--interval", help="Configures syslog rate limit interval in seconds for host", type=click.IntRange(0, 2147483647))
+@click.option("-b", "--burst", help="Configures syslog rate limit burst in number of messages for host", type=click.IntRange(0, 2147483647))
+@clicommon.pass_db
+def rate_limit_host(db, interval, burst):
+    """ Configure syslog rate limit for host """
+
+    if interval is None and burst is None:
+        raise click.UsageError('Either interval or burst must be configured')
+
+    if interval == 0 or burst == 0:
+        click.echo(f'Disable syslog rate limit for host')
+        interval = 0
+        burst = 0
+
+    data = {}
+    if interval is not None:
+        data[SYSLOG_RATE_LIMIT_INTERVAL] = interval
+    if burst is not None:
+        data[SYSLOG_RATE_LIMIT_BURST] = burst
+    db.cfgdb.mod_entry(SYSLOG_CONFIG_TABLE, SYSLOG_CONFIG_GLOBAL_KEY, data)
+
+
+@syslog.command("rate-limit-container")
+@click.argument("service_name", required=True)
+@click.option("-i", "--interval", help="Configures syslog rate limit interval in seconds for containers", type=click.IntRange(0, 2147483647))
+@click.option("-b", "--burst", help="Configures syslog rate limit burst in number of messages for containers", type=click.IntRange(0, 2147483647))
+@clicommon.pass_db
+def rate_limit_container(db, service_name, interval, burst):
+    """ Configure syslog rate limit for containers """
+
+    if interval is None and burst is None:
+        raise click.UsageError('Either interval or burst must be configured')
+
+    features = db.cfgdb.get_table(FEATURE_TABLE)
```

Comment:
should we validate the feature name at the beginning of the function?

Reply:
validating `if interval is None and burst is None:` is cheaper, so I put it to the beginning.

Reply:
@Junchao-Mellanox please add a feature name validator:
```python
def service_validator(ctx, source, service):
```